frontend/src/authenticated/Logout.js

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { useDispatch } from "react-redux";
 import { deleteToken } from "../utils";
+import { logOut } from "../store/actions/actions";
 
-const Logout = ({ setIsLoggedIn }) => {
-  const logOut = () => {
-    setIsLoggedIn(false);
+const Logout = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const logOutHandler = () => {
     deleteToken();
+    dispatch(logOut());
+    history.push("/");
   };
 
-  return (
-    <button onClick={logOut}>log out</button>
-  );
+  return <button onClick={logOutHandler}>log out</button>;
```

Comment:
I think Prettier is still doing that. Please configure that.
```suggestion
  return (
    <button onClick={logOutHandler}>log out</button>
  );
```

Reply:
As agreed this will stay like this for time being.